scripts/setup-ubuntu.sh

```diff
@@ -122,9 +122,16 @@ function install_folly {
   cmake_install -DBUILD_TESTS=OFF
 }
 
+function install_libhdfs3 {
+  github_checkout apache/hawq master
```

Comment:
is libhdfs only part of the hawq project? is it not provided as a standalone library?

Reply:
I did find this: https://github.com/erikmuttersbach/libhdfs3 but it seems to me as not as active as the hawq repo. 

Also from this ticket it looks like for now apache/hawq is where libhdfs3 is being maintained.
https://issues.apache.org/jira/browse/HDFS-6994